### ISSUE TITLE
Ran once ok, 2nd time NameError: name 'utf8' is not defined

### ISSUE THREAD

**Issue by calocedrus** [OzymandiasTheGreat/klembord#2]
`klembord` looks really neat!
But I've ran into this issue:

I've just installed klembord (python 3.6, linux), tried it in a python console:
```
Python 3.6.7 (default, Oct 22 2018, 11:32:17) 
[GCC 8.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import klembord
>>> klembord.init()
>>> klembord.get_text()
'https://pypi.org/project/klembord/\nhttps://www.popular'
```
It copied the content of my clipboard all right, including the line break `\n`.
Within the same python console session, I copy something else to the clipboard (select text, ctrl+c) but it then threw a name 'utf8' not defined error:
```
>>> klembord.get_text()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ludovic/.local/lib/python3.6/site-packages/klembord/__init__.py", line 346, in get_text
    return SELECTION.get_text()
  File "/home/ludovic/.local/lib/python3.6/site-packages/klembord/__init__.py", line 156, in get_text
    return data.decode(utf8)
NameError: name 'utf8' is not defined
```
Why is it so?
I tried again by closing the session, import and init klembord but `get_text()` gave me the exact same error.

I could get my text by setting the content following the example:
```
>>> content = {'UTF8_STRING': 'string'.encode()}
>>> klembord.set(content)
```
then `klembord.get(['UTF8_STRING'])` does return the clipboard.
But why did `get_text` worked a first time and no more on subsequent calls, no matter within the same python session or in a new one?

_Note: looking at the code it seemed indeed `utf8` isn't define, changing to `UTF8` worked..._

**Comment by OzymandiasTheGreat** [comment]
Not sure why it worked the first time, but looking at the traceback, I need to either quote `utf8` or change it to a constant.
Really, should've done more testing before pushing this out.

**Comment by chrahunt** [comment]
Should be fixed by #7.
